src/order/order-request-sender.ts

```diff
@@ -46,7 +46,7 @@ export default class OrderRequestSender {
         });
     }
 
-    submitOrder(body: InternalOrderRequestBody, { headers, timeout }: SubmitOrderRequestOptions = {}): Promise<Response<InternalOrderResponseBody>> {
+    submitOrder(body?: InternalOrderRequestBody, { headers, timeout }: SubmitOrderRequestOptions = {}): Promise<Response<InternalOrderResponseBody>> {
         const url = '/internalapi/v1/checkout/order';
 
         return this._requestSender.post<InternalOrderResponseBody>(url, {
```

Comment:
Do we need to do a check if `body` is undefined here? Or `{ body: undefined }` is a valid payload for the API?

Reply:
`/internalapi/v1/checkout/order` endpoint doesn't actually require any payload. So yes, `{ body: undefined }` is valid.